dkms.in

```diff
@@ -43,7 +43,7 @@ readonly dkms_conf_variables="CLEAN PACKAGE_NAME
 # All of the variables not related to signing we will accept from framework.conf.
 readonly dkms_framework_nonsigning_variables="source_tree dkms_tree install_tree tmp_location
    verbose symlink_modules autoinstall_all_kernels
-   modprobe_on_install"
+   modprobe_on_install parallel_jobs"
```

Comment:
@bodgerer It would be good to also document this variable in `dkms_framework.conf.in`

Reply:
@bodgerer any chance to update? I think it's a good idea, but the request from @zeroepoch is reasonable. If you are not doing it, i will open a separate MR with the same topic. Thanks.